pmm-app/pr.codecept.js

```diff
@@ -38,7 +38,8 @@ exports.config = {
     homePage: './tests/pages/homePage.js',
     remoteInstancesPage: './tests/pages/remoteInstancesPage.js',
     adminPage: './tests/pages/adminPage.js',
-    qanPage: './tests/QAN/qanPage.js',
+    qanPage: './tests/QAN/pages/qanPage.js',
+    qanActions: './tests/QAN/steps/qanActions.js',
```

Comment:
I see the separate Actions page. This is great, but didn't we want to use page fragments? to have separate object for each Component?

Reply:
technically it is the same page object, just that the scope for this task was to break actions and Locators separate and get these tests up and running, we can have another task for page fragments for QAN page.

Reply:
oh, thats clear now for me